ios/Classes/SwiftPolarPlugin.swift

```diff
@@ -76,8 +67,6 @@ public class SwiftPolarPlugin:
   }
 
   public func handle(_ call: FlutterMethodCall, result: @escaping FlutterResult) {
-    initApi()
```

Comment:
Did you make this change just to match the Android SDK? If we still auto-init on method call I think this is fine the way it is.